<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpSetup</class>
 <widget class="QMainWindow" name="TcpSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>381</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);
background-color: rgb(186, 189, 182);</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 215, 207);
font: 15pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 189, 182);
font: 15pt &quot;Ubuntu&quot;;
background-color: rgb(211, 215, 207);</string>
      </property>
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(136, 138, 133);</string>
     </property>
     <property name="text">
      <string>Server_set</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 236);
font: 13pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>192.168.30.222</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 215, 207);
font: 13pt &quot;Ubuntu&quot;;
background-color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>6666</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>750</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Close_Server</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>700</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
